### PR TITLE
Add views and style

### PR DESCRIPTION
**Pull request by JustoRago**
Added content to the views, with embedded ruby code. Added styling to the views according to the wireframe.

### CODE REVIEW

File: app/views/users/show.html.erb [hunk 1]
```diff
@@ -1 +1,30 @@
...
+          <p><%= post.title %></p>
+          <p><%= post.text.truncate(50, separator:' ') %></p>
+          <p class='count'>
+            Comments: <%= post.comments_counter != nil ? post.comments_counter : '0' %>
+            Likes: <%= post.likes_counter != nil ? post.likes_counter : '0' %>
+          </p>
+        </div>
+      <% end %>
+      <a class='button' href=<%= user_posts_url(@user.id)%>>See All Posts</a>
+    </div>
```
Comment (OLIPLICHE) [inline app/views/users/show.html.erb:28]: - Okay 👍 
Capture

File: app/views/users/show.html.erb [hunk 2]
```diff
@@ -1 +1,30 @@
...
+        <div class='leftcard'>
+          <h2 class='username'><%= @user.name %></h2>
+          <p class='count'>Number of posts: <%= @user.posts_counter >= 0 ? @user.posts_counter : '0' %></p><br>
+        </div>
+      </div>
+      <div class='biodiv'>
+        <p>Bio</p>
+        <p><%= @user.bio %></p>
+      </div>
+      <% @posts.each do |post| %>
```
Comment (OLIPLICHE) [inline app/views/users/show.html.erb:17]: - I would advise you to use `#recent_post` method because you need to render the user's most recent post

**Review by OLIPLICHE — CHANGES_REQUESTED** [review]
Hi @asdt560, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- Linter is green (the last commit has a green check mark (✅)) 👏
- README file is descriptive 👏
- Commit messages are meaningful 👏 👏

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: app/views/users/show.html.erb (hunk 2 above)
Comment (JustoRago) [reply app/views/users/show.html.erb:17]: I am using the recent_post method, it is being done in the controller. The @posts variable passed to users/show.html.erb is defined as:

`def show
    @user = User.find(params[:id])
    @posts = @user.recent_posts
  end`

File: app/views/users/show.html.erb (hunk 1 above)
Comment (JustoRago) [reply app/views/users/show.html.erb:28]: If it is ok, what should I change?

**Review by SajjadAhmad14 — APPROVED** [review]
# Status: Approved :heavy_check_mark: :heavy_check_mark: :heavy_check_mark:

### Hi Justo, 
## To Highlight 🎉
No linter errors. :heavy_check_mark:
Git flow is followed. :heavy_check_mark:
## Congratulations 🎉 Thank you for implementing all the requirements 🙏
### Please go ahead and merge the branch!

## Happy Coding 👏👏👏
#### Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
#### Please remember to tag me in your question so I can receive the notification.
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Pull request merged**